### DISCUSSION TITLE
website popup not identified by keepassxc

### DISCUSSION THREAD

**Discussion by riker65** [keepassxreboot/keepassxc#11489] [Q&A] [answered]
Hi
i am trying to get those credentials saved in keepassxc but website credential popup is not recognized.

![image](https://github.com/user-attachments/assets/06b7f337-ba64-48b0-a2ed-fbfa4c9c0280)

Tried to customize it, and also rerecognized credential fields does not solve it for me.
underlying website calling the popup does not have credentials 

This is esphome webserver credential popup

any ideas what am I doing wrong? 

thanks

**Comment by phoerious** [comment, marked as answer]
That's an HTTP basic auth popup. You need to enable "Allow filling HTTP Basic Auth credentials" in your browser extension to fill those. Please be aware that your credentials are not being transmitted securely on this page.
